src/routes/summary/_SummaryItem.svelte

```diff
@@ -29,7 +33,12 @@
 		{/if}
 	</div>
 	{#if time}
-		<div class="summary-time">
+		<div
+			class="summary-time"
+			style="font-family: {$font === 'random'
+				? 'Noto Sans JP'
+				: $font}, sans-serif !important; font-weight: 400 !important;"
+		>
```

Comment:
Same as above, I don't think the summary kana should have a random font. Just use the interface font, M+